Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -341,10 +339,12 @@ class AppStoreConnectService {
     }
 
     func removeTestersFromGroup(groupName: String, emails: [String]) throws {
-        let groupId = try GetBetaGroupOperation(options: .init(app: nil, betaGroupName: groupName))
-            .execute(with: requestor)
-            .await()
-            .id
+        let groupId = try GetBetaGroupOperation(
+            options: .init(appId: nil, bundleId: nil, betaGroupName: groupName)
+        )
+        .execute(with: requestor)
+        .await()
+        .id
```

Comment:
```suggestion
        let groupId = try GetBetaGroupOperation(
                options: .init(appId: nil, bundleId: nil, betaGroupName: groupName)
            )
            .execute(with: requestor)
            .await()
            .id
```

Reply:
I actually don't like adding too many tabs :-) One tab is good enough